.github/workflows/build_test_base_images.yaml

```diff
@@ -53,11 +47,7 @@ jobs:
       - name: Determine Dockerfile
         id: dockerfile
         run: |
-          if [ -f "./docker/test_base/ubuntu-${{ matrix.ubuntu_version }}.Dockerfile" ]; then
-            echo "file=ubuntu-${{ matrix.ubuntu_version }}.Dockerfile" >> "$GITHUB_OUTPUT"
-          else
-            echo "file=Dockerfile" >> "$GITHUB_OUTPUT"
-          fi
+          echo "file=Dockerfile" >> "$GITHUB_OUTPUT"
```

Comment:
Since we drop ubuntu 18.04 test now, all test base images are using the same Dockerfile, does it mean we can just remove the `Determine Dockerfile` step 👀